Makefile

```diff
@@ -49,3 +49,18 @@ origin-release:
 	@echo "  docker push $(IMAGE_ORG)/origin-release:latest"
 	@echo "  docker push $(IMAGE_ORG)/origin-cluster-kube-apiserver-operator"
 	@echo "  OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=$(IMAGE_ORG)/origin-release:latest bin/openshift-install cluster --log-level=debug"
+
+GO_PACKAGES :=./pkg/... ./cmd/...
+
+GO_E2E_PACKAGES :=./e2e/operator/...
+
+.PHONY: check-e2e-operator
+check-e2e-operator:
```

Comment:
try this https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/Makefile#L34-L36

I'd call it just `e2e`. (`check` usually means verify semantics)

Reply:
Done.

Reply:
I meant mainly you should use:
```Makefile
e2e: GO_TEST_PACKAGES :=./test/e2e
e2e: test-unit
.PHONY: e2e
```
and you don't have to duplicate the junit handling logic

Reply:
Done